### ISSUE TITLE
[Bug] [UWP] CollectionView selected state not working on UWP

### ISSUE THREAD

**Issue by Estevete** [xamarin/Xamarin.Forms#9279]
### Description
CollectionView selected state item does not work on UWP
### Steps to Reproduce

1. Create CollectionView with items and add ItemTemplate (StackLayout with Label for example)
2. Add VisualStateManager to the StackLayout and set state `Selected` BackgroundColor red
3. Launch on UWP
4. Launch on iOS to compare

### Expected Behavior
Same behavior than iOS
### Actual Behavior
Doesn't do anything when selected
### Basic Information

Version with issue: 4.5.0.142-pre1
Last known good version: Doesn't work on 4.4.0.991265 either
IDE: Visual Studio 2019 16.4.2
Platform Target Frameworks:
UWP: 18362
Nuget Packages: No unusual ones
Affected Devices: Windows PC

### Screenshots
![image](https://user-images.githubusercontent.com/19273351/72902555-f914cf00-3d2b-11ea-89aa-1e7eb5d53f7d.png)

![image](https://user-images.githubusercontent.com/19273351/72902908-8a844100-3d2c-11ea-9849-d02f01a7edcb.png)

### Reproduction Link
Demo project:
https://github.com/MADSENSE/Madsense.XamarinForms.Sample/tree/CollectionView-selectedState

### Workaround
This can help others:
https://stackoverflow.com/a/59041106/9802022

**Comment by ttmher** [comment]
As a workaround you can add a TapGestureRecognizer to the item and trigger the visual state change manually: 

```
        private SelectedFrame;
        private void OnCollectionItemFrameTapped(object sender, EventArgs args)
        {
            if (SelectedFrame != null)
            {
                VisualStateManager.GoToState(SelectedFrame, "Normal");
            }
            SelectedFrame = (Frame)sender;
            VisualStateManager.GoToState(SelectedFrame, "Selected");
        }
```
Unfortunately this is not working on the first load/start when you have an item selected.

**Comment by samhouts** [comment]
closed by #10770
